pkg/kubectl/service.go

```diff
@@ -136,39 +136,41 @@ func generate(genericParams map[string]interface{}) (runtime.Object, error) {
 		}
```

Comment:
It seems I missed it, will check it and fix, but it certainly appears so. Thanks!